### DISCUSSION TITLE
Does Gitbook support Google Tag Manager (GTM-*) tags?

### DISCUSSION THREAD

**Discussion by waveminded** [GitbookIO/community#823] [0. General bugs and questions]
We want to utilize Google Tag Manager tags in Gitbook (not Google Analytics IDs). Is this possible?
Example: GTM-*

**Comment by aikoii** [comment]
@waveminded 👋 Nope, we don’t support Google Tag Manager integration due to security concerns.

**Reply by waveminded** [reply]
Can you elaborate what security concerns and how they would be different from supporting Google Analytics or the other analytics providers you support?

**Reply by aikoii** [reply]
This is because it allows the injection of CSS/JS, which can often break sites. Additionally, there are security concerns associated with injecting code.
